namespaces/live-1.cloud-platform.service.justice.gov.uk/oasys-keycloak-development/resources/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "domain" {
+  default = "oasys-auth-development.service.justice.gov.uk"
```

Comment:
There are multiple domains in the PR.

You can create one like `oasys-auth.service.justice.gov.uk` and avoid creating multiple domains.